src/main/server.js

```diff
@@ -45,21 +45,17 @@ function extractAndVerifyJwt(req, res, next){
                     return token
                 } catch (err) {
                     console.log({ err })
-                    next(new errors.InvalidCredentialsError('Invalid token'))
                     return null;
                 }
             }
             else {
-                next(new errors.InvalidCredentialsError('Format is Authorization: Bearer [token] or Jwt [token]'))
                 return null
             }
         } else {
-            next(new errors.InvalidCredentialsError('Format is Authorization: Bearer [token] or Jwt [token]'))
             return null
         }
     }
     else {
-        next(new errors.InvalidCredentialsError('Format is Authorization: Bearer [token] or Jwt [token]'))
```

Comment:
I was wondering if this was necessary, glad you removed it. Now `next` isn't used at all, can be removed from the signature and also from `authenticate` 's signature